libCacheSim/cache/eviction/GLCache/README

```diff
@@ -15,9 +13,10 @@ and calculates the segment utility over time using U_seg = \sum \frac{1}{D_obj \
 Note that `D_obj` is the distance/time _since snapshot_. Each object is counted for calculating 
 segment utility, when it has been counted, we mark the `seen_after_snapshot' and does not count again. 
 
-Notce that the X (features) used in training is the features of the 
-snapshotted segments *at the snapshot time*, 
-and the Y (utility) is calculated over time as objects are requested again, 
+Notce that the X (features)
+used in training is the features of the snapshotted segments *at the
+    snapshot time *,
```

Comment:
is the formatting expected?

Reply:
Mistakenly formatted. Have recovered it.